Neos.ContentRepository.Core/Classes/CommandHandlingDependencies.php

```diff
@@ -38,8 +38,10 @@ final class CommandHandlingDependencies
      */
     private array $overriddenContentGraphInstances = [];
 
-    public function __construct(private readonly ContentRepository $contentRepository)
-    {
+    public function __construct(
+        private readonly ContentRepository $contentRepository,
+        private readonly ContentRepositoryReadModel $adapter
```

Comment:
"adapter" is not very specific, what about:
```suggestion
        private readonly ContentRepositoryReadModel $contentRepositoryReadModel
```